src/Resource/CloudAccount/CloudAccountException.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * @package Nexcess-SDK
+ * @license https://opensource.org/licenses/MIT
+ * @copyright 2018 Nexcess.net, LLC
+ */
+
+declare(strict_types  = 1);
+
+namespace Nexcess\Sdk\Resource\CloudAccount;
+
+use Nexcess\Sdk\Exception;
+
+/**
+ * Error conditions for cloud account resources.
+ */
+class CloudAccountException extends Exception {
+
+  /** @var int Invalid download target for a cloud account backup. */
+  const INVALID_PATH = 1;
+
+  /** @var int Target download filename already exists. */
+  const FILE_EXISTS = 2;
+
+  /** @var int Requested cloud account backup was not found. */
+  const BACKUP_NOT_FOUND = 3;
+
+  /** @var int Attempt to access/operate on an invalid backup instance. */
+  const INVALID_BACKUP = 4;
+
+  /** @var int backup file failed to open */
+  const INVALID_STREAM = 5;
```

Comment:
need to add an entry in `INFO` for this code, and a corresponding message in en_US.json

Reply:
done